src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  padding-right: 50px;
+  display: inline-block;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  padding: 22px 0;
+  text-decoration: none;
+  list-style: none;
+  margin: 0 10px;
```

```suggestion
  margin: 0 10px;
```
Зараз у всіх елементів навігації створюються відступи по бокам по 10 пікселів. Але якщо запустити тести, то ми можемо бачити, що у крайніх елеменів (першого і останнього) немає відступів зліва і зправа відповідно. Відступи є тільки МІЖ самим елементами.
Тому необхідно обнулити марджини для першого або останнього елемента, використовуючи :first-child або :last-child